src/commands/serve.ts

```diff
@@ -28,6 +28,15 @@ export class ServeCommand implements Command {
       browser: options.browser,
       componentDir: options['component-dir'],
       packageName: options['package-name'],
+    };
+
+    if (options.env) {
+      console.log(options.env.impl);
+      if (!options.env.impl.serve) {
+        throw new Error(`Serve unimplemented in environment ${options.env.name}`);
```

Comment:
that's why I commented... here you're requiring the environment to implement `serve()`